.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: php
 
 php:
+  - 5.5
```

Comment:
Hi!

Thanks for your pull request. It's a great contribution.

We're unable to support PHP 5.5, so if you can modify this PR to support 5.6 upwards then the CI can pass and we can release this update.

Thanks again

Reply:
@darrynten sorry for delay. I will take a look.

Reply:
@torinaki any luck with this?

Reply:
@darrynten done and I have added a test